checkout/src/main/java/com/checkout/base/model/Country.kt

```diff
@@ -29,8 +29,10 @@ public enum class Country(internal val iso3166Alpha2: String, internal val diali
     BERMUDA("BM", "1441"),
     BHUTAN("BT", "975"),
     BOLIVIA("BO", "591"),
+    BONAIRE("BQ", "5997"),
```

Comment:
@maxim-nosov-cko Are they no unit test for these changes ?

Reply:
@deepesh-vasthimal-cko 
No, there are no unit tests.
What kind of tests do you want to see for the enum class?

Reply:
For iOS we have something like this : https://github.com/checkout/frames-ios/blob/7960a882dc25cf2f84ade3c519f916f32cda3c60/Checkout/Checkout/Test/Model/CountryTests.swift

Reply:
`test_Country_init` is not relevant to current changes, probably a test for fun from() could be written.

The purpose of `test_allAvailable_matchesExpected` is not clear to me at all. With the same success, tests can be written for all constant variables, i.e. value of `some_contant` equals the value of that constant.

In Android we use enum and function [`values()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/enum-values.html) which is a default Koltin functionality, returns array of all enum values. And as far as I'm concerned it's final and can't be even overridden.

Reply:
@deepesh-vasthimal-cko If you still want to see those tests on Android please let me know.